resources/views/components/org-in-list.blade.php

```diff
@@ -1,19 +1,10 @@
-<div
-    x-data="{
-        link: '{{ route('organizations.show', ['organization' => $org->slug]) }}',
-    }"
-    x-on:click="window.location = link"
-    onClick="window.location = this.getAttribute('link')"
-    wire:key="org-{{ $org->id }}"
-    class="group relative cursor-pointer rounded-lg bg-black/4 p-4 backdrop-blur-lg transition duration-300 hover:bg-black/13 md:p-6 md:pt-5"
->
-    <h2 class="mb-5 text-xl font-bold">
-        <img src="{{ $org->favicon }}" alt="{{ $org->name }}" class="mr-2 inline-block w-9 rounded-lg" />
-        {{ $org->name }}
+<div class="group relative cursor-pointer rounded-lg bg-black/4 p-4 backdrop-blur-lg transition duration-300 hover:bg-black/13 md:p-6 md:pt-5">
```

Comment:
Thanks for cleaning this up. There was a lot of cruft left over both from Livewire and also from when the UI was more complicated.

However, in cleaning this up, you can no longer click on the image of the card to get to the page. Can you help with that?

Reply:
I think it was because the image comes later in the DOM... I found two fixes for this:
1. Move the heading to the bottom and flip it with flexbox (I chose this one)
2. Add an extra link that is invisible at the bottom and makes the span inside that one expand

I've chosen the first fix, but let me know if you want me to implement the other one.